python-squid/src/test/resources/docstring.py

```diff
@@ -0,0 +1,33 @@
+"""
+This is a module docstring
+"""
+
+def function_with_docstring():
+	"""This is a function docstring"""
+	pass
+
+def function_without_docstring():
+#   ^^^^^^^^^^^^^^^^^^^^^^^^^^
+	pass
+
+def function_with_empty_docstring():
+#   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+	""" """
+	pass
+
+def one_line_function_with_docstring(): "This is a function docstring"; pass
```

Comment:
@pynicolas Agreed x 2. I had noticed this glitch but I left it aside as the Python way is to use separate lines for docstrings.